README.md

```diff
@@ -1,188 +1,81 @@
-
-The first way is to set the `--db_url` option to the URL endpoint.
+**[Live service](https://normalize.cancervariants.org/gene)**
 
-```commandline
-gene_update --update_all --db_url="http://localhost:8001"
-```
-
-The second way is to set the `GENE_NORM_DB_URL` environment variable to the URL endpoint.
-```commandline
-export GENE_NORM_DB_URL="http://localhost:8001"
-```
-
-To use a PostgreSQL instance instead of DynamoDB, provide a PostgreSQL connection URL instead, e.g.
-
-```commandline
-export GENE_NORM_DB_URL="postgresql://postgres@localhost:5432/gene_normalizer"
-```
-
-### Adding and refreshing data
-
-Use the `gene_update` command in a shell to update the database.
+**[Documentation](https://gene-normalizer.readthedocs.io/en/latest/.html)** · [Installation](https://gene-normalizer.readthedocs.io/en/latest/install.html) · [Usage](https://gene-normalizer.readthedocs.io/en/latest/usage.html) · [API reference](https://gene-normalizer.readthedocs.io/en/latest/api/api.html)
 
-#### Update source(s)
+---
 
-The normalizer currently pulls data from [HGNC](https://www.genenames.org/), [Ensembl](https://useast.ensembl.org/index.html), and [NCBI](https://www.ncbi.nlm.nih.gov/gene/).
+## Install
 
-To update one source, simply set `--sources` to the source you wish to update. The normalizer will check to see if local source data is up-to-date, acquire the most recent data if not, and use it to populate the database.
+The Gene Normalizer is available on [PyPI](https://pypi.org/project/gene-normalizer/):
 
-For example, run the following to acquire the latest HGNC data if necessary, and update the HGNC gene records in the normalizer database:
-
-```commandline
-gene_update --sources="hgnc"
+```shell
+python3 -m pip install gene-normalizer
 ```
 
-To update multiple sources, you can use the `--sources` option with the source names separated by spaces.
+See [installation instruction](https://gene-normalizer.readthedocs.io/en/latest/install.html) in the documentation for a description of installation options and data setup requirements.
 
-#### Update all sources
+## Examples
 
-To update all sources, use the `--update_all` flag:
+Use the [live service](https://normalize.cancervariants.org/gene) to programmatically normalize gene terms, as in the following truncated example:
 
-```commandline
-gene_update --update_all
+```shell
+$ curl 'https://normalize.cancervariants.org/gene/normalize?q=BRAF' | python -m json.tool
+{
+    "query": "BRAF",
+    "match_type": 100,
+    "gene_descriptor": {
+        "label": "BRAF",
+        "gene_id": "hgnc:1097",
+        "xrefs": [
+            "ensembl:ENSG00000157764",
+            "ncbigene:673"
+        ],
+        "alternate_labels": [
+            "BRAF1",
+            "B-RAF1",
+            "NS7",
+            "RAFB1",
+            "B-raf",
+            "BRAF-1"
+        ]
+    }
+    # ...
```

Comment:
Schema has changed for 2x. See [here](https://github.com/cancervariants/gene-normalization/blob/staging/gene/schemas.py#L328) for an example